frontend/components/base.jsx

```diff
@@ -58,7 +58,7 @@ class Base extends React.Component {
         </div>
       );
       break;
-    case "MaryAnn":
+    case "Mary Ann":
       base = (
         <Narration>You find MaryAnn around the cornhole games.</Narration>
```

Comment:
Instead of changing the case shouldn't we change the the Narration text to "Mary Ann"

Adding the space between her name for any text that will be user-facing

Reply:
Needs to have a space in the case or else the maze will not render when they get there.